### PR TITLE
Bug 1814479: bindata/etcd: run etcdctl inside of container

### PR DESCRIPTION
**Pull request by hexfusion**
This PR adds a new bash function exec_etcdctl. this allows for etcdctl commands to be run within the actual container on the host.

### CODE REVIEW

**Comment by hexfusion** [comment]
/hold 

lets do this right and just use the etcdctl container

File: bindata/etcd/etcd-common-tools [hunk 1]
```diff
@@ -20,3 +20,14 @@ function dl_etcdctl {
   podman rm "${etcdctr}"
   etcdctl version
 }
+
+# execute etcdctl command inside of running etcdctl container
+function exec_etcdctl {
+  local command="$@"
+  local container_id=$(sudo crictl ps --label io.kubernetes.container.name=etcdctl -o json | jq -r '.containers[0].id') || true
```
Comment (retroflexer) [inline bindata/etcd/etcd-common-tools]: `sudo` here is redundant. We check earlier that the script should be run with EUID as root.

**Pull request merged**